packages/client/src/v2-events/features/events/useCertificateTemplateSelectorFieldConfig.test.ts

```diff
@@ -0,0 +1,211 @@
+      }
+    ]
+  }
+}
+const declaration = {
+  'mother.firstname': 'Mom',
+  'mother.surname': 'Test',
+  'mother.dobUnknown': true,
+  'mother.age': '80',
+  'mother.nationality': 'FAR',
+  'mother.idType': 'NATIONAL_ID',
+  'mother.nid': '9898989898',
+  'mother.address': {
+    country: 'ZAF',
+    addressType: 'INTERNATIONAL',
+    state: 'Western Cape',
+    district2: 'City Centre',
+    cityOrTown: 'Cape Town'
+  },
+  'mother.maritalStatus': 'SINGLE',
+  'mother.educationalAttainment': 'NO_SCHOOLING',
+  'mother.occupation': '',
+  'father.detailsNotAvailable': true,
+  'father.reason': 'Ran',
+  'informant.relation': 'MOTHER',
+  'informant.phoneNo': '0791234567',
+  'informant.email': 'mom@example.com',
+  'child.firstname': 'Kido',
+  'child.surname': 'Test',
+  'child.gender': 'male',
+  'child.dob': dateToday.toISOString(),
+  'child.reason': 'Procastination',
+  'child.placeOfBirth': 'HEALTH_FACILITY',
+  'child.birthLocation': '278fe332-e9d9-42e9-9ce3-80df8d37e97f',
+  'child.attendantAtBirth': 'PHYSICIAN',
+  'child.birthType': 'SINGLE',
+  'child.weightAtBirth': 2.5
+} as EventState
+
+let mockTemplates: CertificateTemplateConfig[] = []
+describe('useCertificateTemplateSelectorFieldConfig', () => {
+  vi.mock('@client/v2-events/hooks/useAppConfig', () => ({
+    useAppConfig: () => ({
+      certificateTemplates: mockTemplates
+    })
+  }))
+
+  beforeEach(() => {
+    vi.resetModules()
+    vi.clearAllMocks()
+    mockTemplates = []
+  })
+
+  it('should return a certificate as an option if it does not have a conditional and matches event type', async () => {
+    mockTemplates = [
+      { ...birthCertificateTemplateWithNoConditional },
+      { ...deathCertificateTemplateWithNoConditional }
+    ]
+
+    const { useCertificateTemplateSelectorFieldConfig } = await import(
+      './useCertificateTemplateSelectorFieldConfig'
+    )
+    const result = useCertificateTemplateSelectorFieldConfig(
+      'v2.birth',
+      declaration
+    ) as SelectField
+  })
+
+  it('does not include birth-certified-certificate if age is less than a year old', async () => {
+    mockTemplates = [birthCertificateTemplateWithMinimumAge(1)]
+
+    const { useCertificateTemplateSelectorFieldConfig } = await import(
+      './useCertificateTemplateSelectorFieldConfig'
+    )
+
+    const birthResult = useCertificateTemplateSelectorFieldConfig(
+      'birth',
+      declaration
+    ) as SelectField
+
+    expect(birthResult.options).toHaveLength(0)
+  })
+
+  it('returns correct label and required fields in the config', async () => {
+    mockTemplates = [{ ...birthCertificateTemplateWithNoConditional }]
+
+    const { useCertificateTemplateSelectorFieldConfig } = await import(
+      './useCertificateTemplateSelectorFieldConfig'
+    )
+    const result = useCertificateTemplateSelectorFieldConfig(
+      'birth',
+      declaration
+    ) as SelectField
+    expect(result.label).toBeDefined()
+    expect(result.required).toBe(true)
+  })
```

Comment:
```suggestion  
```
There is no logic here that you're testing. Label and required are static fields. There's no point in testing that.